### DISCUSSION TITLE
Make example

### DISCUSSION THREAD

**Discussion by haiviet2011** [sekigon-gonnoc/Pico-PIO-USB#39] [Q&A]
```
$ cmake -G "MinGW Makefiles" ..
PICO_SDK_PATH is C:/VSARM/sdk/pico/pico-sdk
PICO platform is rp2040.
Build type is Release
PICO target board is pico.
Using board configuration from C:/VSARM/sdk/pico/pico-sdk/src/boards/include/boards/pico.h
TinyUSB available at C:/VSARM/sdk/pico/pico-sdk/lib/tinyusb/src/portable/raspberrypi/rp2040; enabling build support for USB.
-- Configuring done
CMake Error at host_hid_to_device_cdc/CMakeLists.txt:2 (add_executable):
  Cannot find source file:

    C:/VSARM/sdk/pico/pico-sdk/lib/tinyusb/src/portable/raspberrypi/pio_usb/dcd_pio_usb.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
  .hpp .hxx .in .txx .f .F .for .f77 .f90 .f95 .f03 .ispc


-- Generating done
```
Hello i try make example getting error thank you.
![image](https://user-images.githubusercontent.com/9393342/175531376-0608d539-2255-4b57-8887-ed79eebe1753.png)

**Comment by TheBrokenEngineer** [comment]
Ah, seems I saw this problem too from the initial clone. I am assuming that you have already run the ``git submodule update --init`` command yes? If not run it now.

Next if the pio_usb directory is still not there then execute the following from the pico-sdk directory.

```git submodule update --remote```

Let me know if this gets you going.

./TBE
